tools/tune/crossover/crossover_build_blob.m

```diff
@@ -0,0 +1,79 @@
+function blob8 = crossover_build_blob(blob_struct, endian)
```

Comment:
have you tested setting the config recently? I am worried the blobs don't align still

Reply:
Maybe you're right. After checking the waveform of 2-way and 4-way modified config, the blobs doesn't seem to align still.
(The output waveform looks weird...)
However I still don't get why the alignment is changed? Because I suppose src/audio/crossover/crossover.c parses coeff blob into iir_state_df2t, which is still the input of current iir_df2t in math library. And both crossover.c and crossover_build_blob.m are un-changed, coeffs should be still aligned?

Reply:
you are right, I appear to be making noise again, apparently I had the common sense to parse our config into the struct to catch struct change errors. Can you post a picture of the waveform?

Reply:
I found the root cause is that there is a mistake on substituting inline biquad function to math library (iir_df2t) for crossover usage. Please see https://github.com/thesofproject/sof/pull/3263/commits/59cd8942844ad41b46eff9616c99a4a333bfeb85